src/FNAPlatform/ModernGLDevice_GL.cs

```diff
@@ -217,11 +217,11 @@ internal enum GLenum : int
 			GL_DEBUG_SEVERITY_NOTIFICATION_ARB =	0x826B
 		}
 
-		// Entry Points
+        // Entry Points
```

Comment:
Tabbing's busted for the first few lines of changes. Also, please put the [Attribute] one line above the delegate declaration.